bin/mozilla/reports.pl

```diff
@@ -5310,7 +5314,7 @@ sub projects_list {
 	$form->{accounttype} = 'standard';
 	while ( my $ref = $sth->fetchrow_hashref(NAME_lc) ) {
 		$form->{link} =
-qq|rp.pl?action=continue&nextsub=generate_projects&projectnumber=$ref->{projectnumber}--$ref->{id}|;
+qq|rp.pl?action=continue&nextsub=generate_projects&fx_transaction=1&projectnumber=$ref->{projectnumber}--$ref->{id}|;
```

Comment:
@NS Can't find that report, but if we add fx, shouldn't it be selectable instead of hardcoded?

Reply:
@sw ns couldn't help me to find that report, do you now?